next.config.js

```diff
@@ -0,0 +1,14 @@
+module.exports = {
+  async rewrites() {
+    if (process.env.NODE_ENV === 'production') {
+      return [];
+    }
+
+    return [
+      {
+        source: '/api/:path*',
+        destination: 'http://localhost:23500/api/:path*',
+      },
+    ];
+  },
+};
```

Comment:
code used to solve CORS access issue when fetching data from backend server. It is a common way to solve the issue when using Next.js